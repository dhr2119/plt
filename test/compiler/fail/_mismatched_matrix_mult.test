int main() {
  matrix int[2,2] m1;
  matrix int[3,3] m2;
  m1 = [1,2;3,4];
  m2 = [1,2,3;4,5,6;7,8,9];
  m2 = m1 * m2;
  return 0;
}
