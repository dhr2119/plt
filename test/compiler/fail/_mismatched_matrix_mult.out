Fatal error: exception Exceptions.MismatchedMatricesForMult("Matrices M1(i1,j1) and M2(i2,j2) must have j1 = i2 and be of same type to be multiplied")
